lib/tanker/identity.rb

```diff
@@ -52,55 +44,61 @@ def self.serialize(hash)
 
     def self.create_identity(b64_app_id, b64_app_secret, user_id)
       assert_string_values({
-        app_id: b64_app_id,
-        app_secret: b64_app_secret,
-        user_id: user_id
-      })
+                             app_id: b64_app_id,
```

Comment:
weird indentation

Reply:
This is to align the arguments with the `assert_string_values({` I think, JetBrains RubyMine does the formatting automatically

Do you want me to force it on the left?

Reply:
whatever, we should just align our auto-linters to not reindent on each pr ;)